common/scripts/kind_provisioner.sh

```diff
@@ -165,6 +165,12 @@ function setup_kind_cluster() {
     echo "No existing kind cluster with name ${NAME}. Continue..."
   fi
 
+  # Create the docker network beforehand so "kind create cluster" doesn't create it with ipv6
+  echo "Deleting previous KinD docker network"
+  docker network rm kind || true
+  echo "Creating the docker network for kind"
+  docker network create -d=bridge -o com.docker.network.bridge.enable_ip_masquerade=true kind
+
```

Comment:
This change will be overridden when the `update-common` make target is run. Because we copy the entire file from the upstream repo: https://github.com/istio/common-files. So, make sense to push this change to the upstream repository and then update using the make target?